src/jit/gentree.cpp

```diff
@@ -9336,7 +9337,20 @@ void Compiler::gtDispNode(GenTree* tree, IndentStack* indentStack, __in __in_z _
             case GT_CALL:
                 if (tree->gtFlags & GTF_CALL_INLINE_CANDIDATE)
                 {
-                    printf("I");
+                    if (tree->gtCall.gtCallMoreFlags & GTF_CALL_M_GUARDED_DEVIRT)
```

Comment:
Isn't this just `IsGuardedDevirtualizationCandidate()`?

Reply:
Yes, if I had a `GenTreeCall`

Reply:
Am I missing something? You do have `tree->gtCall`

Reply:
Good point.